### PR TITLE
Refresh README

### LINKED ISSUE #153: Adjust readme
I think we need a proper readme here, with the brief description of the package.

### LINKED ISSUE #334: Fix badges in `README`
The `{pkgdown}` attributes are included in `README`. They should actually be included in `.Rmd`, and not `.md` file.

image

### PR DESCRIPTION
**Pull request by IndrajeetPatil**
Closes #153
Closes #334

### CODE REVIEW

**Comment by IndrajeetPatil** [comment]
You only need to review `README.Rmd`, and not `README.md`.

**Comment by Yuri05** [comment]
@IndrajeetPatil Why do we need to generate README.md from README.Rmd?
The advantage of Rmd is R code execution during the generation process - but here we don't have any R code to be executed...

**Comment by IndrajeetPatil** [comment]
How can you solve #334 without using `.Rmd`?

**Comment by IndrajeetPatil** [comment]
I mean now the issue is solved, but it wouldn't have been without the Rmd. 

So I can delete it now, but, in general, I prefer generating md from Rmd.

**Comment by Yuri05** [comment]
> How can you solve #334 without using `.Rmd`?

Is the pkgdown Intro page generated from README.md or from README.Rmd?
If from .Rmd - don't know how to solve it without :)

**Review by PavelBal — APPROVED** [review]

**Comment by IndrajeetPatil** [comment]
Yeah, I have removed the .Rmd file now.

File: README.md [hunk 1]
```diff
@@ -1,43 +1,72 @@
-# TLF Library
 
-TLF Library implementation in R
+<!-- README.md is generated from README.Rmd. Please edit that file -->
```
Comment (Yuri05) [inline README.md:2]: let's remove the comment as well

Comment (IndrajeetPatil) [reply README.md]: Done.

**Review by Yuri05 — APPROVED** [review]

**Pull request merged**